fixtures/large/rspec_mocks_proxy_expected.rb

```diff
@@ -414,25 +416,23 @@ def original_method_handle_for(message)
 
       def original_unbound_method_handle_from_ancestor_for(message)
         double = method_double_from_ancestor_for(message)
-        double && double.original_method.unbind
+        double&.original_method&.unbind
       end
 
       def method_double_from_ancestor_for(message)
         @method_doubles.fetch(message) do
           # The fact that there is no method double for this message indicates
           # that it has not been redefined by rspec-mocks. We need to continue
           # looking up the ancestor chain.
-          return superclass_proxy && superclass_proxy.method_double_from_ancestor_for(message)
+          return superclass_proxy&.method_double_from_ancestor_for(message)
         end
       end
 
       def superclass_proxy
         return @superclass_proxy if defined?(@superclass_proxy)
 
-        if (superclass = object.superclass)
```

Comment:
it is correct for standard to make this change, it's not a change rubyfmt can make